lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -111,7 +111,7 @@ oslo.utils==3.41.0
 packaging==19.0
 paramiko==2.5.0
 parsley==1.3
-paste==3.0.8
+paste==3.0.8 ; python_version == '2.7'
```

Comment:
This is wrong, it was fixed in commit fed80f19f6340f01401390c1ef5850e1fd71153a, you need to re-run `make update-dependencies` now that you have merged the upstream `dev` branch.

Reply:
@nsoranzo Thanks, should be fixed now.